guillotina/contrib/redis/dm.py

```diff
@@ -42,7 +69,8 @@ async def _save(self):
         redis = await self.get_redis()
         key = self.get_key()
         self._data["last_activity"] = time.time()
-        await redis.set(key, json.dumps(self._data, cls=GuillotinaJSONEncoder), expire=self._ttl)
+        with watch("set"):
+            await redis.set(key, json.dumps(self._data, cls=GuillotinaJSONEncoder), expire=self._ttl)
```

Comment:
Should the `json.dumps` be outside the contextmanager?

Reply:
good idea. I will move it